modules/crm/includes/functions-customer.php

```diff
@@ -1035,7 +1044,7 @@ function erp_crm_send_schedule_notification( $activity, $extra = false ) {
             array_push( $users, $created_user );
 
             foreach ( $users as $key => $user ) {
-                $body = sprintf( __( 'You have a schedule after %s %s at %s', 'erp' ), $extra['notification_time_interval'], $extra['notification_time'], date( 'F j, Y, g:i a', strtotime( $activity['start_date'] ) ) );
+                $body = sprintf( __( 'You have a schedule after %s %s at %s%s', 'erp' ), $extra['notification_time_interval'], $extra['notification_time'], date( 'F j, Y, g:i a', strtotime( $activity['start_date'] ) ), empty( $extra['client_time_zone'] ) ? '' : ( '(' . $extra['client_time_zone'] . ')' ) );
```

Comment:
Replace date with `erp-current_datetime`.
Also, the indexes should be checked whether set or not.


```
isset( $extra['notification_time'], $extra['notification_time_interval'] )
```